### ISSUE TITLE
DeprecationWarnings

### ISSUE THREAD

**Issue by powellquiring** [paramiko/paramiko#2135]
I'm not using paramiko directly.  Running pytest of my code generated:

```
============================================================================================== warnings summary ===============================================================================================
py/test_transit.py: 64 warnings
  /Users/xxxx/github.com/YYYY/vpc-transit/venv/lib/python3.10/site-packages/paramiko/transport.py:420: DeprecationWarning: setDaemon() is deprecated, set the daemon attribute instead
    self.setDaemon(True)

py/test_transit.py: 64 warnings
  /Users/xxxx/github.com/YYYY/vpc-transit/venv/lib/python3.10/site-packages/paramiko/channel.py:1069: DeprecationWarning: notifyAll() is deprecated, use notify_all() instead
    self.out_buffer_cv.notifyAll()

py/test_transit.py: 801 warnings
  /Users/xxxx/github.com/YYYY/vpc-transit/venv/lib/python3.10/site-packages/paramiko/buffered_pipe.py:104: DeprecationWarning: notifyAll() is deprecated, use notify_all() instead
    self._cv.notifyAll()

py/test_transit.py: 256 warnings
  /Users/xxxx/github.com/YYYY/vpc-transit/venv/lib/python3.10/site-packages/paramiko/buffered_pipe.py:206: DeprecationWarning: notifyAll() is deprecated, use notify_all() instead
    self._cv.notifyAll()

py/test_transit.py: 64 warnings
  /Users/xxxx/github.com/YYYY/vpc-transit/venv/lib/python3.10/site-packages/paramiko/channel.py:1233: DeprecationWarning: notifyAll() is deprecated, use notify_all() instead
    self.out_buffer_cv.notifyAll()

```

If you need a better cut down let me know.

**Comment by powellquiring** [comment]
Wait, looking more closely I see that there is a constraint on the version being used:

```
paramiko==2.7.2
```

**Comment by bskinn** [comment]
This looks like a problem in some code that's calling into paramiko, using an outdated interface. You'll need to track down which tool/package is using paramiko internally, and report it there, I'm pretty sure.

**Comment by jun66j5** [comment]
The same has been filed in #1838 and fixed in paramiko 2.9.4.

**Comment by bskinn** [comment]
Ahh, right, I remember that issue now, thanks @jun66j5.
